content/tutorials/manage-the-rippled-server/troubleshooting/diagnosing-problems.md

```diff
@@ -112,6 +115,43 @@ It is normal for a `rippled` the server to print many warning-level (`WRN`) mess
 For a more thorough explanation of various types of log messages, see [Understanding Log Messages](understanding-log-messages.html).
 
 
+## Info Collection Script
+
+If you have problems diagnosing the problem, or you are unable to resolve the problem with any of the common fixes, you may want to ask for help in a support forum or the [GitHub issues](https://github.com/ripple/rippled/issues). When asking for help, you can use an info collection script to gather information about your system to help others diagnose the issue.
+
+The official package installation (for [Ubuntu/Debian](install-rippled-on-ubuntu.html) or [CentOS/RedHat](install-rippled-on-centos-rhel-with-yum.html)) installs such a script by default, to `/opt/ripple/bin/getRippledInfo`. [New in: rippled 1.5.0][] If you compiled `rippled` yourself, you can find the same script [in the `rippled` source code repository](https://github.com/ripple/rippled/blob/develop/bin/getRippledInfo).
+
+To use the script:
+
+1. Run the script while `rippled` is running.
+
+        $ /opt/ripple/bin/getRippledInfo
+
+        ####################################################
+          rippled info has been gathered. Please copy the
+          contents of /tmp/ripple_info.Xo8Xr/rippled_info.md
+          to a github gist at https://gist.github.com/
+
+          PLEASE REVIEW THIS FILE FOR ANY SENSITIVE DATA
+          BEFORE POSTING! We have tried our best to omit
+          any sensitive information from this file, but you
+          should verify before posting.
+        ####################################################
+
+    The script collects the output of numerous commands and writes them to a temporary file. The filename is randomized with a string of letters and numbers (case-sensitive), for example: `/tmp/ripple_info.Xo8Xr/rippled_info.md`
+
+
+2. Look over the output file for sensitive information.
+
+    Although the script attempts to scrub sensitive information from the information it collects, it's important to look it over to be sure before posting the output publicly. For example, the script outputs detailed information about your server hardware, and you may want to remove some sections for privacy reasons. Use a text editor to read the output file and to remove anything you don't want to post.
```

Comment:
Look over is problematic, especially for English as a 2nd language (too similar to overlook). Why not use something more active like the following, and also state what they might want to do with sensitive info they locate. 

View the output file for any sensitive information you may want to remove before you post.

The pronoun "it" in the second half of the first sentence is a bit confusing. Might want to restate that they should are "view the generated output file",  as "it" could be misinterpreted as the script itself. 

How does the script attempt to scrub sensitive information? Does it replace base endpoints with variables? Does it remove host ports?